packages/core/contracts/optimistic-assertor/implementation/examples/Insurance.sol

```diff
@@ -44,20 +45,28 @@ contract Insurance {
         address payoutAddress,
         bytes memory insuredEvent
     ) public returns (bytes32 policyId) {
-        policyId = keccak256(abi.encode(insuredEvent, payoutAddress));
-        require(policies[policyId].insurer == address(0), "Policy already exists");
+        bytes memory timestampedEvent =
+            abi.encodePacked(
+                "Insurance contract is claiming that insurance event ",
+                insuredEvent,
+                " occured after timestamp ",
```

Comment:
did you mean "before" as one should not claim payout unless the insurance event has already occurred.